migration/process-migration/src/main/java/io/camunda/migration/process/adapter/MigrationRepositoryIndex.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migration.process.adapter;
+
+import io.camunda.webapps.schema.descriptors.operate.OperateIndexDescriptor;
+
+/* Fork of Operate's MigrationRepositoryIndex descriptor for correct prefixing */
+public class MigrationRepositoryIndex extends OperateIndexDescriptor {
```

Comment:
❓ Should we move this to the webapps-schema?

Reply:
Do we have to? I think not, for greenfield 8.7 deployments you won't have to do migration and for existing ones this index would be already present and it won't be used again after 8.7.